web/src/js/__tests__/urlStateSpec.js

```diff
@@ -0,0 +1,79 @@
+import initialize from '../urlState'
+
+import reduceFlows from '../ducks/flows'
+import reduceUI from '../ducks/ui/index'
+import reduceEventLog from '../ducks/eventLog'
+import * as flowsActions from '../ducks/flows'
+
+import configureStore from 'redux-mock-store'
+
+const mockStore = configureStore()
+
+describe('updateStoreFromUrl', () => {
+    history.replaceState = jest.fn()
```

Comment:
` history.replaceState = jest.fn()` is not necessary, it should be deleted.
But the initial state is necessary, because `urlStare.js` only export `initialize()` function, every time we invoke `updateStoreFromUrl`, `updateStoreFromUrl` will be invoked later, so if we omit these initial states, it will throw a exception like: ` TypeError: Cannot read property 'filter' of undefined` from `updateUrlFromStore`.